src/VisualStudio/TestUtilities/Remoting/RemotingHelper.cs

```diff
@@ -28,6 +29,14 @@ public static InteractiveWindowWrapper CreateCSharpInteractiveWindowWrapper()
             return new InteractiveWindowWrapper(vsInteractiveWindow.InteractiveWindow);
         }
 
+        private static WorkspaceWrapper CreateWorkspaceWrapper()
+        {
+            var componentModel = (IComponentModel)(ServiceProvider.GlobalProvider.GetService(typeof(SComponentModel)));
```

Comment:
Avoid using ServiceProvider.GlobalProvider if you can. Do we have a better context for this object we can use? Do we already know if the package has been loaded?